src/docs/api/authentication/guide_oauth-service/index.md

```diff
@@ -33,24 +33,29 @@ Create an OAuth Service App when you want to write a service application that ma
 1. Click **New** to display the **Create Application** dialog box. Select the option to create a new app and then select the **Service** app type and give the app a name.
 1. On the **App Configuration** page, in the Service Account field, select the appropriate service principal that the application will use and **save** the changes.
 1. Take note of the auto-generated Client ID.
-1. On the **Authentication** tab, create a new Access Key. There are two types of Access Keys you can create for the client credentials flow.
-
-   - Choose create a public **access key** if the service app uses the [Laserfiche API client libraries](../../libraries/) or is capable of generating an HMAC client credential.
-   - Choose create a long-lasting **authorization key** if the service app does not have the capabilities of generating an HMAC client credential. This option is intended for [low-code tools](../../../getting-started/guide_low-code-tools-v1/) that require a static secret to connect to Laserfiche APIs.
-   - Creating an **authorization key** requires entering the service principal key for the selected service principal in the - - {: .note } **Note:** The authorization key will be invalidated when the service principal key expires or is rotated. A new authorization key will need to be created using a valid service principal key.
+1. On the **Authentication** tab in the **OAuth 2.0 Scopes** section, select the allowed scopes for your application. These are the scopes that you will be able to request when obtaining an access token. See [OAuth 2.0 Scopes for Laserfiche APIs](../guide_oauth_2.0_scopes/) for more details.
+1. On the **Authentication** tab, create a new Access Key. There are three types of Access Keys you can create. Two require the use of the OAuth client credentials flow, and the last generates a username and password you can use directly with the APIs.
+
+   - Choose create a public **access key** if the service app uses the [Laserfiche API client libraries](../../libraries/) or is capable of generating an HMAC client credential. Once generated, continue to the [OAuth Client Credentials Grant Flow](#oauth-client-credentials-grant-flow) section to generate an HMAC client credential and request an access token.

```

Comment:
Use the Authentication tab to generate the access key most suitable to your use case. Service applications support the following options:

Option 1: Using a **Public 'Access Key' with a 'Service Principal Key'** pair to obtain the API Access Token. BLA-BLA 
Option 2: Using a long-lasting **Authorization Key** to obtain the API Access Token. BLA-BLA  
Option 3: Using a username and password to directly access the API. BLA-BLA

Reply:
![image](https://github.com/Laserfiche/laserfiche.github.io/assets/99926015/d587a6b0-95b8-4b15-a73c-25300b02bf27)